lib/Common/Core/StackBackTrace.cpp

```diff
@@ -34,19 +34,40 @@ ULONG StackBackTrace::Capture(ULONG framesToSkip)
 size_t
 StackBackTrace::Print()
 {
-    DbgHelpSymbolManager::EnsureInitialized();
-
     size_t retValue = 0;
 
+#ifdef DBGHELP_SYMBOL_MANAGER
+    DbgHelpSymbolManager::EnsureInitialized();
+
     for(ULONG i = 0; i < this->framesCount; i++)
     {
         PVOID address = this->stackBackTrace[i];
         retValue += Output::Print(_u(" "));
         retValue += DbgHelpSymbolManager::PrintSymbol(address);
         retValue += Output::Print(_u("\n"));
     }
+#else
+    char** f = backtrace_symbols(this->stackBackTrace, this->framesCount);
```

Comment:
This shouldn't be decoding function names on xplat with current config. Actually, addresses are also useless there (we compile with PIC...) anyway, we take care of this later.